woocommerce/woocommerce-android
# [Manual Taxes M#1] Tax rates education dialog

Closes: #9591 


⚠️ Based on changes in FluxC: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2812

### Description

This PR adds "info" icon next to "Taxes Total" label in order creation. When the icon is clicked the taxes information dialog is displayed.

### Testing instructions

Go to order creation or order edition screen
1. Add products to the order
2. Click on the "?" icon next to "Taxes" label
3. The dialog should be displayed
4. Verify it redirects to wp-admin tax rates setting page

### Images/gif


<img width=300 src=https://github.com/woocommerce/woocommerce-android/assets/4527432/a96f9fa9-ce68-409e-b995-a29c5236b68e/>
